Workflow/Sources/AnyWorkflowConvertible.swift

```diff
@@ -147,3 +147,18 @@ struct OutputBlockingWorkflow<Child: AnyWorkflowConvertible>: Workflow {
             .rendered(in: context)
     }
 }
+
+// MARK: -
+
+extension AnyWorkflowConvertible where Output == Never {
```

Comment:
i think this extension should be moved near [these existing ones](https://github.com/square/workflow-swift/blob/deb7e21e0c97fe4c9010d338efb24440e44e69a9/Workflow/Sources/AnyWorkflow.swift#L189).